src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -347,38 +346,37 @@ private void downloadJavaFXZip(String osarch) throws IOException {
                 "javafxStaticSdk",
                 configuration.getJavafxStaticSdkVersion(),
                 configuration.getTargetTriplet().getOsArch());
-
-        Logger.logDebug("Process zip javafx done");
+        Logger.logInfo("JavaFX static libs downloaded successfully");
     }
 
     /**
      * Crafts Android SDK url and then downloads it 
      * @throws IOException in case anything goes wrong.
      */
     private void downloadAndroidSdkZip() throws IOException {
+        Logger.logInfo("Downloading Android SDK...");
```

Comment:
Add that this might take a long time, and that you can press CTRL-C to stop the process.

Reply:
We show the total size of the file and the current size that's being downloaded so far. Adding another line to tell that this will take a long time sounds redundant to me.